File: 1. Begginer/9- Pida al usuario dos números reales (que se guardarán en las variables dato1 y dato2) y muestre en pantalla el resultado de dividir dato1 entre dato2/main.go
package main
import "fmt"

func main() {
	/* Another way to declarate variables in GoLang */
	var (
		data1 float32
		data2 float32
	)
	fmt.Println("Ingrese un número real: ")
	fmt.Scanf("%f", &data1)
	fmt.Println("Ingrese otro número real: ")
	fmt.Scanf("%f", &data2)
	fmt.Println("El resultado de dividir", data1, " entre ", data2, " es igual a ", data1/data2)
}

File: 1. Begginer/9- Pida al usuario dos números reales (que se guardarán en las variables dato1 y dato2) y muestre en pantalla el resultado de dividir dato1 entre dato2/main.c
#include<stdio.h>

int main() {
	float data1;
	float data2;
	printf("Ingrese el primer número real: ");
	scanf("%f", &data1);
	printf("Ingrese el segundo número real: ");
	scanf("%f", &data2);
	printf("El resultado de dividir %f entre %f es igual a %f", data1, data2, data1/data2);
}